presto-spi/src/main/java/io/prestosql/spi/type/Chars.java

```diff
@@ -60,6 +60,34 @@ public static Slice padSpaces(Slice slice, int length)
         return buffer;
     }
 
+    /**
+     * Pads String with spaces to given length in code points.
+     * <p>
+     * Note: unlike {@code com.google.common.base.Strings#padEnd(java.lang.String, int, char)},
+     * this respects code points encoded as UTF-16 surrogate pairs.
+     */
+    public static String padSpaces(String s, CharType charType)
+    {
+        int length = charType.getLength();
```

Comment:
The type isn't used to anything other than getting the length to pad to. Pass the length to this method, instead (it would also make it consistent with the javadoc above)

Reply:
Initially i had `int length`, but I thought this might be ambiguous (it requires reading the javadoc to know what the method is doing, even though it looks simple & obvious).
Also, with `int length` I would prefer not to static-import this (but that's not enforceable)

Reply:
i updated the javadoc